RC: Supported Cloud providers regions (#334)
redis/docs: content/operate/rc/supported-regions.md

---
Title: Supported Cloud providers and regions
alwaysopen: false
categories:
- docs
- operate
- rc
description: null
linktitle: Supported regions
weight: 90
---

> Choosing your cloud provider and region may affect latency between your application and your database, and may affect what connectivity options are available for your database.

Reviewer: ```suggestion
Your choice of cloud provider and region may affect latency between your application and your database, and may affect what connectivity options are available for your database.
```

Redis Cloud supports databases on the following cloud providers:
- [Amazon Web Services (AWS)](#amazon-web-services)
- [Google Cloud](#google-cloud)
- [Microsoft Azure](#microsoft-azure)

## Amazon Web Services

> Redis Cloud supports databases on all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 

Reviewer: This suggestion might not be correct if a "region" is understood to mean a server more than a geographical region.
```suggestion
Redis Cloud supports databases in all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
```

Author: ```suggestion
Redis Cloud supports databases in the following Amazon Web Services (AWS) regions. 
```

| `ap-southeast-2` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
| `ap-southeast-3` | Jakarta, Indonesia   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `ap-southeast-4` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |

### Middle East and Africa

| Region ID      | Location                | Redis Cloud Pro | Redis Cloud Essentials |
|----------------|-------------------------|-----------------|------------------------|
| `af-south-1`   | Cape Town, South Africa | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `il-central-1` | Tel Aviv, Israel        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `me-central-1` | UAE                     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `me-south-1`   | Bahrain                 | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |

## Google Cloud

> Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 

Reviewer: Same as above.

> Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 

Reviewer 2: Same comment as in AWS. "All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. " is not necessary IMO

Author: ```suggestion
Redis Cloud supports databases in the following Google Cloud regions. 
```

> 

Reviewer 2: @cmilesb redundant parenthesis at the end of the sentence

> Redis Cloud Pro databases on Google Cloud support [VPC Peering]({{< relref "/operate/rc/security/vpc-peering#gcp-vpc-peering" >}}) and [Private Service Connect]({{< relref "/operate/rc/security/private-service-connect" >}})).

Author: ```suggestion
Redis Cloud Pro databases on Google Cloud support [VPC Peering]({{< relref "/operate/rc/security/vpc-peering#gcp-vpc-peering" >}}) and [Private Service Connect]({{< relref "/operate/rc/security/private-service-connect" >}}).
```

| `asia-south1`          | Mumbai, India        | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
| `asia-south2`          | Delhi, India         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `asia-southeast1`      | Singapore            | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `asia-southeast2`      | Jakarta, Indonesia   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `australia-southeast1` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
| `australia-southeast2` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |

### Middle East

| Region ID  | Location         | Redis Cloud Pro | Redis Cloud Essentials |
|------------|------------------|-----------------|------------------------|
| `me-west1` | Tel Aviv, Israel | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |

> ## Microsoft Azure

Reviewer 2: Can we put more clear dividers (maybe also bigger font) between the cloud providers? It looks pretty similar to the dividers between the different locations insider each provider.

Author: I made the location headings smaller, let me know what you think.

Redis Cloud Essentials is available on the following Microsoft Azure regions:

| Region ID | Location        |
|-----------|-----------------|
| `east-us` | Virginia, USA   |
| `west-us` | California, USA |

> Redis Cloud Pro is available on Azure through Azure Cache for Redis Enterprise. See [Azure Cache for Redis pricing](https://azure.microsoft.com/en-us/pricing/details/cache/) to view the list of Azure regions that support Azure Cache for Redis Enterprise.

Reviewer 2: not bolded enough. Maybe inside of a note?

Author: I put this part inside of a note, let me know what you think.
